src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -228,6 +229,19 @@ private static void addAwsConfigGroup(final ConfigDef configDef) {
             AWS_S3_BUCKET_NAME_CONFIG
         );
 
+        configDef.define(
+            AWS_S3_SSEA_NAME_CONFIG,
+            Type.STRING,
+            null,
+            new ConfigDef.NonEmptyString(),
+            Importance.MEDIUM,
+            "AWS S3 Server Side Encryption Algorithm",
```

Comment:
docs: add sample valid values -- even though there's no strict values to validate, let's hint users what values are expected.